targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/PeripheralNames.h

```diff
@@ -0,0 +1,48 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2018 Arm Limited
```

Comment:
new files should have 2019

Reply:
All files should be 20XX-2019? what is the starting year?

Reply:
New files: 2019
Older files: X-Y   Y is updated to 2019

Reply:
Done